guides/demo.md

```diff
@@ -0,0 +1,170 @@
+
+## How to setup Wanda in demo mode?
+
+First set up the [local environment](./development/hack_on_wanda.md) and run the following command to start Wanda in demo mode, instead of the usual `iex -S mix phx.server`
+
+```bash
+$ DATABASE_URL=ecto://postgres:postgres@localhost:5434/wanda_dev \
+  AMQP_URL=amqp://wanda:wanda@localhost:5674 \
+  SECRET_KEY_BASE=Tbp26GilFTZOXafb7FNVqt4dFQdeb7FAJ++am7ItYx2sMzYaPSB9SwUczdJu6AhQ \
+  CORS_ORIGIN=http://localhost:4000 \
+  ACCESS_TOKEN_ENC_SECRET=s2ZdE+3+ke1USHEJ5O45KT364KiXPYaB9cJPdH3p60t8yT0nkLexLBNw8TFSzC7k \
+  PORT=4001 \
+  MIX_ENV=demo \
+  iex -S mix phx.server
+```
+
+### Explanation of environment variables:
+
+- DATABASE_URL: The URL to connect to the Postgres database running in the Docker container.
+- AMQP_URL: The URL for RabbitMQ, used for message exchange.
+- SECRET_KEY_BASE: The secret key for Wanda.
+- CORS_ORIGIN: The origin URL from where API requests are allowed (pointing to web).
+- ACCESS_TOKEN_ENC_SECRET: Secret key for encrypting access tokens.
+- PORT: The port on which Wanda will run (4001 in this case).
+- MIX_ENV: The environment in which Wanda will run (set to "demo" for the demo environment).
+
+## Modify demo facts configuration
+
+In the Wanda demo environment, configure fake facts by editing the [fake_facts configuration](/priv/demo/fake_facts.yaml). Define custom facts for different targets and checks.
+
+### YAML Structure
+
+The Configuration is saved in [fake_facts.yaml](/priv/demo/fake_facts.yaml) file, which follows a specific structure with two main sections: targets and facts.
+
+Example:
+
+```yaml
+targets:
+  target1: 0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4
+  target2: 13e8c25c-3180-5a9a-95c8-51ec38e50cfc
+facts:
+  check_1:
+    fact_name1:
+      target1: 2
+      target2: 3
+```
+
+#### Targets
+
+The targets section allows defining handles for targets' UUIDs, providing a reference that can be used through the configuration.
+
+Format:
+
+```yaml
+targets:
+  <target_reference>: <target_id>
+```
+
+<target_reference>: A user-defined handle or reference for a target.
+
+<target_id>: The UUID or identifier of the target.
+
+#### Add a new target to the demo configuration
+
+Open the [fake_facts](priv/demo/fake_facts.yaml) configuration and add a new target entry to the targets section, following this format:
+
+```
+targetX: <new_target_id>
+```
+
+Replace targetX with an unique target identifier (e.g target3) and <new_target_id>
+
+Add new target with an identifier (e.g target3) and a new target id:
+
+```
+targets:
+  target1: 0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4
+  target2: 13e8c25c-3180-5a9a-95c8-51ec38e50cfc
+  target3: <new_target_id>
+```
+
+### Facts Section:
+
+The facts section allows specifying the facts associated with specific targets and checks. Each fact corresponds to a piece of information that a target can return during the evaluation of a check.
```

Comment:
> Each fact corresponds to a piece of information that a target can return during the evaluation of a check.
>
This is not totally correct. Facts are what targets return from facts gathering, not during the evaluation.
Evaluation is something that happens after fact gathering, which uses gathered facts in evaluation.

```suggestion
The facts section allows specifying what fact value a target should return for a specific check.
```